packages/webdriverio/src/commands/element/execute.ts

```diff
@@ -0,0 +1,63 @@
+import { verifyArgsAndStripIfElement } from '../../utils/index.js'
+
+/**
+ *
+ * Inject a snippet of JavaScript into the page for execution in the context of the currently selected
+ * frame using the given element as scope, because it is on the element scope it means that WebdriverIO will
+ * automatically wait for the element to exist before executing the script.
+ * The executed script is assumed to be synchronous and the result of evaluating the script is returned to
+ * the client.
+ *
+ * The script argument defines the script to execute in the form of a function body. The value returned by
+ * that function will be returned to the client. The function will be invoked with the provided args array
+ * and the values may be accessed via the arguments object in the order specified.
+ *
+ * Arguments may be any JSON-primitive, array, or JSON object. JSON objects that define a WebElement
+ * reference will be converted to the corresponding DOM element. Likewise, any WebElements in the script
+ * result will be returned to the client as WebElement JSON objects.
+ *
+ * <example>
+    :execute.js
+    it('should wait for the example element to exist, then inject javascript on the page', async () => {
+        const result = await $('example').execute((...args, elem) => {
+            console.log(args) // logs added arguments
+            console.log(elem) // logs the element
+            return a + b + c + d
```

Comment:
I wonder if we could connect the example to an element here, e.g. show an html element like:

```html
<div>Hello World!</div>
```
and the example be like:
```ts
const text = await $('div').execute((elem, a, b, c, d) => {
  return elem.textContent + a + b + c + d
}, 1, 2, 3, 4);
console.log(text); // returns "Hello World1234"
```